pkg/operator/resources/operator/operator.go

```diff
@@ -456,15 +458,304 @@ func createVMImport() *extv1beta1.CustomResourceDefinition {
+																	Description: `ResourceMappingItem defines the mapping of a single resource from the provider to kubevirt`,
+																	Properties: map[string]extv1beta1.JSONSchemaProps{
+																		"source": {
+																			Description: `Source defines how to identify a resource on the provider, either by ID or by name`,
+																			Type:        "object",
+																			Properties: map[string]extv1beta1.JSONSchemaProps{
+																				"id": {
+																					Type: "string",
+																				},
+																				"name": {
+																					Type: "string",
+																				},
+																			},
+																		},
+																		"target": {
+																			Description: `ObjectIdentifier defines how a resource should be identified on kubevirt`,
+																			Type:        "object",
+																			Properties: map[string]extv1beta1.JSONSchemaProps{
+																				"name": {
+																					Type: "string",
+																				},
+																				"namespace": {
+																					Type: "string",
+																				},
+																			},
+																			Required: []string{"name"},
+																		},
+																		"type": {
+																			Type: "string",
+																		},
+																	},
+																	Required: []string{"source", "target"},
+																},
+															},
+														},
+													},
+												},
+												"vm": {
+													Type:        "object",
+													Description: `VirtualMachineImportOvirtSourceVMSpec defines the definition of the VM info in oVirt`,
+													Properties: map[string]extv1beta1.JSONSchemaProps{
+														"cluster": {
+															Description: `VirtualMachineImportOvirtSourceVMClusterSpec defines the source cluster's identity of the VM in oVirt`,
+															Type:        "object",
+															Properties: map[string]extv1beta1.JSONSchemaProps{
+																"id": {
+																	Type: "string",
+																},
+																"name": {
+																	Type: "string",
+																},
+															},
+														},
+														"id": {
+															Type: "string",
+														},
+														"name": {
+															Type: "string",
+														},
+													},
+												},
+											},
+											Required: []string{"vm"},
+										},
+									},
+								},
+								"startVm": {
+									Type: "boolean",
+								},
+								"targetVmName": {
+									Type: "string",
+								},
+							},
+							Required: []string{"providerCredentialsSecret", "source"},
+						},
+						"status": {
+							Type:        "object",
+							Description: `VirtualMachineImportStatus defines the observed state of VirtualMachineImport`,
+							Properties: map[string]extv1beta1.JSONSchemaProps{
+								"conditions": {
+									Description: "A list of current conditions of the VMImportConfig resource",
```

Comment:
I think we can change `VMImportConfig` to `VirtualMachineImport`

Reply:
good catch